src/sisl/geom/flat.py

```diff
@@ -204,3 +204,33 @@ def graphene_flake(
     if atoms is None:
         atoms = Atom(Z=6, R=bond * 1.01)
     return honeycomb_flake(shells, bond, atoms, vacuum)
+
+
+@set_module("sisl.geom")
+def triangulene(n: int, bond: float = 1.42, atoms=None) -> Geometry:
```

Comment:
I think users will expect `n` to be the first argument, since the rest should default to "relaxed" graphene. But I don't know how bad it is to do `triangulene(n=3)`, probably not that bad, it still feels natural.

Reply:
I agree with you @pfebrer , only one parameter is strictly needed. So yes, this would be annoying.

Reply:
Ok!